client/extensions/woocommerce/app/order/create.js

```diff
@@ -131,18 +155,35 @@ export default connect(
 		const siteId = site ? site.ID : false;
 		const orderId = getCurrentlyEditingOrderId( state );
 		const isSaving = isOrderUpdating( state, orderId );
-		const hasOrderEdits = ! isEmpty( getOrderEdits( state ) );
+		const orderEdits = getOrderEdits( state );
+		const hasOrderEdits = ! isEmpty( omit( orderEdits, [ 'currency' ] ) );
```

Comment:
Maybe add a comment here too, about why you need to omit currency?

Reply:
Good idea! Fixed in https://github.com/Automattic/wp-calypso/pull/23568/commits/8a7c403776596e9294f02baf3bf29b7b5c04d140